packages/editor/src/client/locales/en.json

```diff
@@ -391,7 +391,7 @@
       "panels": {
         "addTag": "Press enter to add tag",
         "attribution": "Attribution",
-        "author": "Author",
+        "author": "Source",
```

Comment:
I am not sure I understand correctly, but do you want to rename the key as well? 👀

Reply:
I did so @bishoymelek